src/Chunk.cpp

```diff
@@ -2600,6 +2600,31 @@ void cChunk::GetBlockInfo(int a_RelX, int a_RelY, int a_RelZ, BLOCKTYPE & a_Bloc
 
 
 
+
+bool cChunk::GetChunkAndRelByAbsolute(const Vector3d & a_Position, cChunk ** a_Chunk, int & a_RelX, int & a_RelY, int & a_RelZ)
+{
+	return GetChunkAndRelByAbsolute(Vector3i(FloorC(a_Position.x), FloorC(a_Position.y), FloorC(a_Position.z)), a_Chunk, a_RelX, a_RelY, a_RelZ);
+}
+
+
+
+
+
+bool cChunk::GetChunkAndRelByAbsolute(const Vector3i & a_Position, cChunk ** a_Chunk, int & a_RelX, int & a_RelY, int & a_RelZ)
```

Comment:
>I used a vector for the new addition because it makes sense

To further elaborate: entities have a vector already, no sense in bisecting it every time we want to query cChunk.